_troubleshoot/index.md

```diff
@@ -60,7 +60,7 @@ The operating system for each OpenSearch node handles encryption of data at rest
 cryptsetup luksFormat --key-file <key> <partition>
 ```
 
-For full documentation on the command, see [the Linux man page](https://man7.org/linux/man-pages/man8/cryptsetup.8.html).
+For full documentation on the command, see [the Linux manual page for cryptsetup](https://man7.org/linux/man-pages/man8/cryptsetup.8.html).
```

Comment:
I think the link name should reflect the title of the destination page:

```
Refer to [cryptsetup(8) — Linux manual page](https://man7.org/linux/man-pages/man8/cryptsetup.8.html) for more information about `cryptsetup`.
```

Reply:
Source: https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md#links

Reply:
Can do. Thanks.